app/components/ui/LandingPageEligibilityBlock.jsx

```diff
@@ -1,46 +1,40 @@
-import React, { Component } from 'react';
+import React from 'react';
 import { Link } from 'react-router';
 import PropTypes from 'prop-types';
 import { eligibilitiesMapping } from '../../utils/refinementMappings';
 import 'url-search-params-polyfill';
 
-class LandingPageEligibilityBlock extends Component {
-
-  buildBlockUrl(eligibility) {
-    var search_params = new URLSearchParams();
-    const eligibilityElements = eligibilitiesMapping[eligibility];
-    eligibilityElements.forEach((element, index) => {
-      search_params.append(`refinementList[eligibilities][${index}]`, element);
-    });
-    return '/search?'+search_params.toString();
-  }
+const buildBlockUrl = eligibility => {
+  const search_params = new URLSearchParams();
+  const eligibilityElements = eligibilitiesMapping[eligibility];
+  eligibilityElements.forEach((element, index) => {
+    search_params.append(`refinementList[eligibilities][${index}]`, element);
+  });
+  return `/search?${search_params.toString()}`;
+};
 
-  render() {
-    // TODO Properly implement horizontal scroll buttons if we ever get > 6 eligibilities
-    return (
-      <div className="landing-page-eligibility-block">
-        <div className="landing-page-eligibility-block__resources">
-          <h2 className="landing-page-eligibility-block__resources__title">
+const LandingPageEligibilityBlock = ({ eligibilities }) => (
+  <div className="landing-page-eligibility-block">
+    <div className="landing-page-eligibility-block__resources">
+      <h2 className="landing-page-eligibility-block__resources__title">
             Discover resources by eligibility
-          </h2>
-          {/* <a className="scroll-button scroll-button-left">&lt;</a> */}
-          <div className="landing-page-eligibility-block__resources-scroller">
-            { this.props.eligibilities.map(eligibility => (
-              <LandingPageCard
-                key={eligibility.id}
-                name={eligibility.name}
-                count={eligibility.service_count}
-                url={this.buildBlockUrl(eligibility.name)}
-              />
-            ))
-            }
-          </div>
-          {/* <a className="scroll-button scroll-button-right">&gt;</a> */}
-        </div>
+      </h2>
+      {/* <a className="scroll-button scroll-button-left">&lt;</a> */}
```

Comment:
Should the "Properly implement horizontal scroll buttons" comment be preserved / moved here if this other comment is also being preserved?

Reply:
Yeah, sorry, I probably got overzealous deleting and moving around code. I'll add the comment back here.